openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -181,6 +183,7 @@ def _create_from_course(cls, course):
             effort=CourseDetails.fetch_about_attribute(course.id, 'effort'),
             course_video_url=CourseDetails.fetch_video_url(course.id),
             self_paced=course.self_paced,
+            instructor_info=json.dumps(course.instructor_info),
```

Comment:
No need to dump to a JSON string here. You can just assign the `instructor_info` dict.